Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectEnergyTransfer.py

```diff
@@ -27,6 +27,13 @@ def _elems_or_none(l):
     return l if len(l) != 0 else None
 
 
+def add_missing_elements(from_list, to_list):
+    for element in from_list:
+        if element not in to_list:
+            to_list.append(element)
+    return sorted(to_list)
```

Comment:
This function is equivalent to this, which you might prefer?
```
to_list.extend(x for x in from_list if x not in to_list)
return sorted(to_list)
```
or for extremely long lists, I believe this is faster
```
to_list + list(set(from_list) - set(to_list))
return sorted(to_list)
```